src/main/sensors/boardalignment.c

```diff
@@ -64,51 +64,55 @@ static void alignBoard(int32_t *vec)
 
 void alignSensors(const int32_t *src, int32_t *dest, uint8_t rotation)
 {
-    static uint32_t swap[3];
-    memcpy(swap, src, sizeof(swap));
+    int32_t x;
```

Comment:
Maybe it would be better to use temporaries:
```
int32_t x = src[X];
int32_t y = src[Y];
int32_t z = src[Z];
```
and then use simple assignment. Variables must be read into registers anyway. Maybe even removing unnecessary `static` from `swap[]` would generate identical/optimal code.